### ISSUE TITLE
Update documentation for String.format capabilities

### ISSUE THREAD

**Issue by aimyth** [robtweed/qewd-transform-json#13]
The current syntax also supports String.format capabilities as it replaces the value found in the mustache variable.

So instead of using functions for string formats we could just use the format capabilities

name: "{{firstName}} {{lastName}}",
This would provide the value
name: "John Doe"
Without having a custom function

**Comment by robtweed** [comment]
Yes, you can have as many {{xx}} substitutions within a value transform.

**Comment by aimyth** [comment]
Thanks for the clarification, can you update your documentation for this?

The samples that you have are all misleading at least to me.

<pre>
`  var myFn = function(input) {
      return 'xxxxxx ' + input + ' yyyyyyy';
  };`
</pre>

one could just write this without the function by
`field: "xxxxxx {{input}} yyyyyyy",`

The function can be for higher level things that can't be done by these simple constructs

**Comment by robtweed** [comment]
Yes, you're right: I do need to update the documentation.  The ability to handle multiple substitutions was added after the documentation was written, so you are correct: a function is now unnecessary
